skt/runner.py

```diff
@@ -582,7 +582,7 @@ def __jobsubmit(self, xml):
 
         return jobid
 
-    def run(self, url, max_aborted, release, wait=False, host=None,
```

Comment:
Could you please remove the `test_run_host` test as well?

Reply:
Didn't notice that one, thanks for finding it!